package.json

```diff
@@ -30,7 +30,7 @@
     "react-native-webview": "*"
   },
   "devDependencies": {
-    "@types/node": "^17.0.23",
+    "@types/node": "^18.19.31",
```

Comment:
This is missing

Reply:
That's the highest available [version](https://www.npmjs.com/package/@types/node/v/18.11.18) for that library.

Reply:
That's weird, isn't it? 😂

Reply:
Also, the version specified on the npm package page is not the same as this one 🤣 ❓